res/css/focus.css

```diff
@@ -21,3 +20,6 @@ button:focus {
     0 0 0 4px var(--blue-50-a30);
   outline: 0;
 }
+a:focus {
+  box-shadow: 0 0 0 2px var(--blue-50), 0 0 0 6px var(--blue-50-a30);
```

Comment:
nit: please add `outline: 0` here, otherwise we see the default outline and it's disgraceful.